src/snowcli/cli/snowpark/commands.py

```diff
@@ -449,7 +448,7 @@ def _replace_handler_in_zip(
     handler_module, _, handler_function = handler.rpartition(".")
     with TemporaryDirectory() as temp_dir:
         wrapper_file = os.path.join(temp_dir, "snowpark_coverage.py")
-        utils.generate_snowpark_coverage_wrapper(
+        snowcli.utils.file_utils.generate_snowpark_coverage_wrapper(
```

Comment:
`import snowcli.utils.file_utils`

so `file_utils.generate_snowpark_coverage_wrapper(` should be enough

Reply:
Right. Done